Backpack/src/androidTest/java/net/skyscanner/backpack/badge/BpkBadgeTest.kt

```diff
@@ -53,17 +55,16 @@ class BpkBadgeTest {
   }
 
   @Test
-  @Ignore // FIXME: Test is not working anymore (the component looks fine)
   fun test_alpha_default() {
     val badge = BpkBadge(context)
     badge.type = BpkBadge.Type.Success
-    Assert.assertEquals(PixelFormat.OPAQUE, badge.background.opacity)
+    Assert.assertEquals(context.getColorStateList(R.color.bpkGlencoe), (badge.background as GradientDrawable).color)
   }
 
   @Test
   fun test_alpha_outline() {
     val badge = BpkBadge(context)
     badge.type = BpkBadge.Type.Outline
-    Assert.assertEquals(PixelFormat.TRANSLUCENT, badge.background.opacity)
+    Assert.assertEquals(ColorStateList.valueOf(Color.TRANSPARENT), (badge.background as GradientDrawable).color)
```

Comment:
also changed this test as the original test would always be true - no matter the type